### ISSUE TITLE
Build error on HHVM 3.6-dev

### ISSUE THREAD

**Issue by steelbrain** [vipsoft/hhvm-ext-geoip#17]
```
root@zoom:/tmp/hhvm-ext-geoip# make
Scanning dependencies of target geoip
[100%] Building CXX object CMakeFiles/geoip.dir/ext_geoip.cpp.o
In file included from /usr/include/hphp/runtime/base/base-includes.h:24:0,
                 from /tmp/hhvm-ext-geoip/ext_geoip.cpp:21:
/usr/include/hphp/runtime/ext/extension.h:26:34: fatal error: hphp/runtime/version.h: No such file or directory
 #include "hphp/runtime/version.h"
                                  ^
compilation terminated.
make[2]: *** [CMakeFiles/geoip.dir/ext_geoip.cpp.o] Error 1
make[1]: *** [CMakeFiles/geoip.dir/all] Error 2
make: *** [all] Error 2
root@zoom:/tmp/hhvm-ext-geoip# hhvm --version
HipHop VM 3.6.0-dev (rel)
Compiler: heads/master-0-g4d9b5b19afc0581856c79298f0672d7b55df9339
Repo schema: dd4735ca46a0f3085a288777a8b8be4fca156b07
```

**Comment by robocoder** [comment]
Hmm.  I thought this was fixed already.

https://github.com/facebook/hhvm/wiki/DSO-3.5.0

**Comment by steelbrain** [comment]
@sgolemon said it was fixed in v3.6.0-dev, funny thing I am on the same version. But nevermind. Thanks!
